### PR TITLE
Removed fixes to enable TLS v1.3

### LINKED ISSUE #1773: Drop fixes for old golang versions
Files in question:
https://github.com/loadimpact/k6/blob/master/lib/options_tls_go1_12.go
https://github.com/loadimpact/k6/blob/master/lib/tls13_version_go1_11.go

### PR DESCRIPTION
**Pull request by codebien**
Removed the functions to enable TLS1.3 on Go 1.11 and 1.12

Closes #1773

### CODE REVIEW

**Review by na-- — CHANGES_REQUESTED** [review]
Thanks! Please also completely remove the `TLSVersion13` constant everywhere in the k6 code and use the Go stdlib's `tls.VersionTLS13` one. For example here: https://github.com/loadimpact/k6/blob/253ad5406a85d222b1d30fb563ef63b32cc7e9a3/lib/tlsconfig.go#L28-L33

File: lib/options.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func (v *TLSVersions) UnmarshalJSON(data []byte) error {
 }
 
 func (v *TLSVersions) isTLS13() bool {
```
Comment (na--) [inline lib/options.go:94]: I think we can safely delete this whole method now.

**Review by na-- — DISMISSED** [review]
Thanks, LGTM besides the minor nitpick I noted as an inline comment.

**Comment by codebien** [comment]
Done, method removed

**Review by na-- — APPROVED** [review]

**Review by MStoykov — APPROVED** [review]

**Pull request merged**